hugo/content/tags/css/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: CSS
+description: CSS stands for Cascading Style Sheets· CSS is the language we use to style a Web page.
```

Comment:
I'm not a fan of your description, But I might be biased

Reply:
updated the description 😊